pyproject.toml

```diff
@@ -59,7 +59,7 @@ pyowm = { version = "^3.3.0", optional = true }
 googlemaps = { version = "^4.10.0", optional = true }
 requests_oauthlib = { version = "^1.3.1", optional = true }
 unstructured = { extras = ["all-docs"], version = "^0.10.30", optional = true }
-
+pillow = { version = "^10.2.0", optional = true }
```

Comment:
why we need this library?

Reply:
Because we need to determine if the input is an image or not in vlm embedding class.

Reply:
also add it under [tool.poetry.extras] **tools** and **all**  part, as well as [[tool.mypy.overrides]]